### PR TITLE
chore: add maintainers as codeowners to all files

### LINKED ISSUE #1012: Add Repo Maintainers to CODEOWNERS file
# Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Description**:

Update this repositories `.github/CODEOWNER` file to ensure that the repository maintainer
group(s) have full control of the files in this repository.

### PR DESCRIPTION
**Pull request by PavelSBorisov**
**Description**:
Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Related issue(s)**:

Fixes #1012 

**Notes for reviewer**:
N/A

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -20,22 +20,22 @@
...
 
 # Codacy Tool Configurations
-/config/                                        @hashgraph/release-engineering-managers
-.remarkrc                                       @hashgraph/release-engineering-managers
+/config/                                        @hashgraph/release-engineering-managers @hashgraph/release-engineering
+.remarkrc                                       @hashgraph/release-engineering-managers @hashgraph/release-engineering
 
 # Semantic Release Configuration
-.releaserc                                      @hashgraph/release-engineering-managers
+.releaserc                                      @hashgraph/release-engineering-managers @hashgraph/release-engineering
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:31]: Add `@hashgraph/devops-ci` to the `.releaserc` file.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -20,22 +20,22 @@
 # NOTE: Must be placed last to ensure enforcement over all other rules
 
 # Protection Rules for Github Configuration Files and Actions Workflows
-/.github/                                       @hashgraph/release-engineering-managers @hashgraph/devops-ci
-/.github/workflows/                             @hashgraph/release-engineering-managers @hashgraph/devops-ci
+/.github/                                       @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/release-engineering
+/.github/workflows/                             @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/release-engineering
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:24]: Confirm with @rbarkerSL but I think `release-engineering` should be removed from the `workflows` folder.

**Review by andrewb1269hg — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (rbarkerSL) [reply .github/CODEOWNERS:24]: Not on this one.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -20,22 +20,22 @@
...
 # Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
 /CODEOWNERS                                     @hashgraph/release-engineering-managers
 
 # Protect the repository root files
 /README.md                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci
-**/LICENSE                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers
+**/LICENSE                                      @hashgraph/release-engineering-managers
 
 # CodeCov configuration
 **/codecov.yml                                  @hashgraph/release-engineering @hashgraph/release-engineering-managers
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:41]:
```suggestion
**/codecov.yml                                  @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/devops-ci-committers 
```

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

**Review by andrewb1269hg — APPROVED** [review]

**Review by rbarkerSL — APPROVED** [review]

**Pull request merged**